src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -46,6 +46,32 @@ public LinuxOperatingSystem() {
         this.version = (osRelease != null) ? new LinuxOSVersionInfoEx(osRelease) : new LinuxOSVersionInfoEx();
     }
 
+    @Override
+    public long getOpenDescriptors() {
+        return getDescriptors(0);
+    }
+
+    @Override
+    public long getMaxDescriptors() {
+        return getDescriptors(2);
+    }
+
+    private long getDescriptors(int index) {
+        if (new File("/proc/sys/fs/file-nr").exists()) {

```

Comment:
I like doing the exists check rather than relying on catching exceptions.  I wonder if we should modify the `FileUtil.readFile()` method to include this check, and catch the exceptions in the util method so all the other implementations can simply call the one method...